de.nicokimmel.shadowcast-electron.json

```diff
@@ -0,0 +1,51 @@
+{
+  "app-id": "de.nicokimmel.shadowcast-electron",
```

Comment:
`nicokimmel.de` seems to be parked. is this yours?

Reply:
> `nicokimmel.de` seems to be parked. is this yours?

It is indeed mine. There is just nothing to show. Has been there like forever since I'm only running services. Do have to create a "product page"?